python/pyspark/errors_doc_gen.py

```diff
@@ -43,7 +43,7 @@ def generate_errors_doc(output_rst_file_path: str) -> None:
 
 This is a list of common, named error classes returned by PySpark which are defined at `error_classes.py <https://github.com/apache/spark/blob/master/python/pyspark/errors/error_classes.py>`_.
 
-When writing PySpark errors, developers must use an error class from the list. If an appropriate error class is not available, add a new one into the list. For more information, please refer to `Contributing Error and Exception <https://spark.apache.org/docs/latest/api/python/development/contributing.html#contributing-error-and-exception>`_.
+When writing PySpark errors, developers must use an error class from the list. If an appropriate error class is not available, add a new one into the list. For more information, please refer to `Contributing Error and Exception <contributing.rst#contributing-error-and-exception>`_.
```

Comment:
Just a question. Did you generate the document with this PR, @itholic ?

Reply:
Yes, I confirmed that the link works properly with this PR as below:

<img width="741" alt="Screenshot 2023-12-09 at 2 18 41 PM" src="https://github.com/apache/spark/assets/44108233/6d42a046-184e-4b62-9c9d-34b3734e096f">